src/components/toolbar/toolbar.js

```diff
@@ -56,12 +56,17 @@ angular.module('material.components.toolbar', [
  * at one fourth the rate at which the user scrolls down. Default 0.5.
  */
 
-function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate ) {
-  var translateY = angular.bind(null, $mdUtil.supplant, 'translate3d(0,{0}px,0)' );
+function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate) {
+  var translateY = angular.bind(null, $mdUtil.supplant, 'translate3d(0,{0}px,0)');
 
   return {
     restrict: 'E',
+    template: '<div ng-transclude></div>',
```

Comment:
Will move the handler.

Re: `isDefined`, my understanding is that if the attribute is present, we want the functionality. So, if they just do
`<md-toolbar md-scroll-shrink>`, we should treat that as `<md-toolbar md-scroll-shrink="true">`.

If we test the scope, though, the first case will be undefined, but we still want to run it.

I will add some comments to make this more clear.